styles/config/vocabularies/coco/accept.txt

```diff
@@ -27,7 +27,7 @@ LLM[s]
 OCI
 oras
 PCR
-Peerpod
+[pP]eer[Pp]od[s]
```

Comment:
nit: my preference would be to not introduce the new variations but make the PR additions to use `Peerpod` consistently.